src/components/Notification/InactivityMessage.jsx

```diff
@@ -62,27 +60,43 @@ const InactivityMessage = () => {
     };
   }, []);
 
-  // Event handler for logout and removing items from localStorage
-  // Returns user to home page upon successful logout
-  // TODO: In future PR, add countdown timer to automatically log user out if they do not select continue
-  // (e.g. "You will be automatically logged out in 5:00 minutes")
-  const handleLogout = () => {
-    localStorage.clear();
-  };
+  /* Starts a five minute timer that is displayed to the user when the 
+  inactivity window pops up. If said timer runs out, the user is 
+  forcibly logged out. */
+  useEffect(() => {
+    if (showPopup) {
+      logoutTimer.current = setInterval(() => {
+        if (secondsToLogout > 0) {
+          setSecondsToLogout((prev) => prev - 1);
+        } else if (secondsToLogout === 0) {
+          logout();
```

Comment:
Think we could clear the localStorage after log out since there's no need for PASS-related items after that. It was part of the handleLogout handler function previously.

Other than that and needing to update the branch, think it's ready to be merged.

Reply:
Cool, I've done both